### DISCUSSION TITLE
Question: Drag and Drop Parent and Children

### DISCUSSION THREAD

**Discussion by sccardais** [vslinko/obsidian-outliner#516] [Q&A]
I have a node / line formatted as H3. Indented below this node, I have several "child" nodes.
The hotkeys CMD+Shift+Up or Down arrows move the child nodes but they don't move the parent and child nodes.

Drag and Drop also works on the child nodes but not the parent.

Is this because the "parent" is formatted as a heading? Is the node formatted as a heading not considered part of the outline?

Is there another way to rearrange or reposition the Parent and its children to a different location in the note besides cutting and pasting?

Thanks for any suggestions.

**Comment by ferileho** [comment]
try TAB and  SHIFT+TAB
